package.json

```diff
@@ -99,10 +100,10 @@
     "eslint-plugin-react": "7.19.0",
     "eslint-plugin-react-hooks": "^2.3.0",
     "i18next-scanner": "2.11.0",
-    "jest": "^26.0.1",
```

Comment:
Was getting a conflict between `react-scripts` and this version of jest when starting the webapp locally - `react-scripts` depends on version ~24. Removed my node_modules and this line, ran `npm install`, and the tests in place still run and pass!